### ISSUE TITLE
Breadcrumbs accepts accessibilityLabel but doesn't use it

### ISSUE THREAD

**Issue by theodoretan** [Shopify/polaris-react#2251]
## Issue summary

The `Breadcrumbs` component accepts an `accessibilityLabel` option but doesn't use it to render any visually hidden text for screen readers.

![image](https://user-images.githubusercontent.com/9055413/66338360-f45c1480-e90e-11e9-9119-470d5afa4b26.png)

Searching for accessibilityLabel:
![image](https://user-images.githubusercontent.com/9055413/66338651-911eb200-e90f-11e9-9c72-926a9f70a8d8.png)

## Expected behavior

`accessibilityLabel` should be used if it's allowed to be passed in.

## Actual behavior

`accessibilityLabel` is not used and there is no hidden text for screen readers.

## Steps to reproduce the problem

1. Render a page with Breadcrumbs and pass in accessibilityLabel
